sql/postgres/migrate.go

```diff
@@ -421,34 +421,43 @@ func (m *migrate) fks(b *sqlx.Builder, fks ...*schema.ForeignKey) {
 	})
 }
 
-// skipAutoDropped filters unnecessary changes that are automatically
-// dropped by the database before running ALTER TABLE.
-func skipAutoDropped(changes []schema.Change) []schema.Change {
+// Build instantiates a new builder and writes the given phrase to it.
+func Build(phrase string) *sqlx.Builder {
+	b := &sqlx.Builder{QuoteChar: '"'}
+	return b.P(phrase)
+}
+
+// skipAutoChanges filters unnecessary changes that are automatically
+// happened by the database when ALTER TABLE is executed.
+func skipAutoChanges(changes []schema.Change) []schema.Change {
 	dropC := make(map[string]bool)
 	for _, c := range changes {
 		if c, ok := c.(*schema.DropColumn); ok {
 			dropC[c.C.Name] = true
 		}
 	}
+search:
```

Comment:
no possibility for an infinite loop? maybe add max counter?

Reply:
`continue` is not `goto`

https://play.golang.org/p/3PzJ51D5rkZ